com.boxy_svg.BoxySVG.yaml

```diff
@@ -0,0 +1,108 @@
+        dest: main
+      - type: file
+        path: com.boxy_svg.BoxySVG.desktop
+        dest: main
+      - type: file
+        path: icons/16x16.png
+        dest: main/icons
+      - type: file
+        path: icons/22x22.png
+        dest: main/icons
+      - type: file
+        path: icons/24x24.png
+        dest: main/icons
+      - type: file
+        path: icons/32x32.png
+        dest: main/icons
+      - type: file
+        path: icons/48x48.png
+        dest: main/icons
+      - type: file
+        path: icons/64x64.png
+        dest: main/icons
+      - type: file
+        path: icons/96x96.png
+        dest: main/icons
+      - type: file
+        path: icons/128x128.png
+        dest: main/icons
+      - type: file
+        path: icons/256x256.png
+        dest: main/icons
+      - type: file
+        path: icons/512x512.png
+        dest: main/icons
+      - type: file
+        path: icons/scalable.svg
+        dest: main/icons
+    build-commands:
+      # Install NPM dependencies
+      - npm install --offline
+      - npm install --offline --prefix ./src
+      # Build the app
+      - |
+        . ../flatpak-node/electron-builder-arch-args.sh
+        npm run dist -- $ELECTRON_BUILDER_ARCH_ARGS  --linux --dir
+      # Bundle app and dependencies
+      - cp -a dist/linux*unpacked /app/main
+      # Install app wrapper
+      - install ../boxy-svg.sh ${FLATPAK_DEST}/bin/boxy-svg
+      # Install metadata files
+      - install -Dm644 com.boxy_svg.BoxySVG.appdata.xml $FLATPAK_DEST/share/appdata/$FLATPAK_ID.appdata.xml
+      - install -Dm644 com.boxy_svg.BoxySVG.desktop     $FLATPAK_DEST/share/applications/$FLATPAK_ID.desktop
+      - install -Dm644 icons/16x16.png                  $FLATPAK_DEST/share/icons/hicolor/16x16/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/22x22.png                  $FLATPAK_DEST/share/icons/hicolor/22x22/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/24x24.png                  $FLATPAK_DEST/share/icons/hicolor/24x24/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/32x32.png                  $FLATPAK_DEST/share/icons/hicolor/32x32/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/48x48.png                  $FLATPAK_DEST/share/icons/hicolor/48x48/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/64x64.png                  $FLATPAK_DEST/share/icons/hicolor/64x64/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/96x96.png                  $FLATPAK_DEST/share/icons/hicolor/96x96/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/128x128.png                $FLATPAK_DEST/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/256x256.png                $FLATPAK_DEST/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/512x512.png                $FLATPAK_DEST/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png
+      - install -Dm644 icons/scalable.svg               $FLATPAK_DEST/share/icons/hicolor/scalable/apps/$FLATPAK_ID.svg
```

Comment:
you only need svg at that point.

Reply:
Do you mean I could get rid of only 128x128.png, 256x256.png and 512x512.png or can I remove all PNG icons and keep only the SVG version?

Reply:
Flatpak require 128x128. But with only the SVG it's fine to omit as well. Everything else can be extrapolated from the SVG.

Reply:
It is not clear to me which icon sizes are required by individual desktop environments and I don't really have the time to research that. Therefore, I would prefer to keep all the icon sizes.

Reply:
that's why SVG exist. Given that the icons are not un the upstream assets.

Reply:
I can see many other apps which are including both the SVG version and PNG version in different sizes. There must be some reason why they are doing it.

Reply:
either I didn't review them or it happened after, or a bit of both. But a proper flatpak package doesn't even need to have any icon, nor a .deskop, nor the appdata here because the upstream source come with them.

SVG + 128x128 is all that needed for icon. Even just either ought to be enough but both are great choice.

Reply:
I removed all icons. The SVG and the 128x128 PNG icons are now retrieved from the upstream package. I did the same with .desktop and .appdata.xml files.